jerry-main/main-unix.c

```diff
@@ -50,6 +50,25 @@
 
 static uint8_t buffer[ JERRY_BUFFER_SIZE ];
 
+#ifdef JERRY_PORT_ENABLE_EXTERNAL_CONTEXT
```

Comment:
Yes, main unix is a demo not a unit test. I don't think anybody wants to compile main-unix with different options, this is not its purpose.